src/extension.ts

```diff
@@ -79,9 +81,14 @@ export function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(compileWebView);
 
+  // run onecc with cfg
+  let runCfgDisposal = vscode.commands.registerCommand('onevscode.run-cfg', (oneNode: OneNode) => {
+    handleRunOnecc(oneNode.node.uri, logger);
+  });
+  context.subscriptions.push(runCfgDisposal);
```

Comment:
How about moving this subscription-registration inside OneExplorer constructor?

Reply:
008bf842aba34205196cd934c79b2b5de3134614

Reply:
Thanks. Fixed as you proposed. 😸